### PR TITLE
ENH: Add ionization mode flag to fingerprint

### LINKED ISSUE #43: Add parameter for ionization mode
Should likely allow users to specify if they want to let sirius guess the mode, or if they want to explicitly set to positive/negative mode.

### PR DESCRIPTION
**Pull request by ElDeveloper**
Fixes #43

### CODE REVIEW

File: q2_chemistree/_fingerprint.py [hunk 1]
```diff
@@ -88,8 +89,16 @@ def fingerprint(sirius_path: str, features: MGFDirFmt, ppm_max: int,
         os.environ['_JAVA_OPTIONS'] = (os.environ.get('_JAVA_OPTIONS', '') +
                                        ' ' + java_flags)
 
+    # qiime2 will check that the only possible modes are positive, negative or
```
Comment (anupriyatripathi) [inline q2_chemistree/_fingerprint.py]: looks good! just need to add unit tests

**Comment by ElDeveloper** [comment]
This needs to be updated with the changes from #46

File: q2_chemistree/tests/test_fingerprint.py [hunk 2]
```diff
@@ -52,6 +52,24 @@ def test_fragmentation_trees(self):
...
+
+    def test_fragmentation_trees_exception(self):
+        ions = self.ions.view(MGFDirFmt)
+        with self.assertRaises(ValueError):
+            compute_fragmentation_trees(sirius_path=self.goodsirpath,
+                                        features=ions,
+                                        ppm_max=15,
+                                        profile='orbitrap',
+                                        ionization_mode='n3gativ3')
+
```
Comment (anupriyatripathi) [inline q2_chemistree/tests/test_fingerprint.py]: XD

Comment (ElDeveloper) [reply q2_chemistree/tests/test_fingerprint.py]: 😆

**Comment by anupriyatripathi** [comment]
Awesome! 👍

**Pull request merged**